src/main/java/mdlaf/components/textfield/MaterialComponentField.java

```diff
@@ -151,6 +154,11 @@ protected synchronized void firePropertyChange(String propertyName, Object oldVa
         propertyChangeSupport.firePropertyChange(propertyName, oldValue, newValue);
     }
 
+    @Override
+    protected void paintSafely(Graphics g) {
+        super.paintSafely(MaterialDrawingUtils.getAliasedGraphics(g));
```

Comment:
I can not accept at the moment this change here, how discussed in the #154.

Reply:
Change reverted in 8483f322af75080cda91b4df1279a5ff8eb30086